controllers/module_reconciler.go

```diff
@@ -397,3 +404,12 @@ func (r *ModuleReconciler) SetupWithManager(mgr ctrl.Manager, kernelLabel string
 		Named("module").
 		Complete(r)
 }
+
+func isNodeSchedulable(node *v1.Node) bool {
+	for _, taint := range node.Spec.Taints {
```

Comment:
Who is responsible for tainting the node?

Reply:
master nodes , that are not schedulable ( not SNO) are tainted by KubeAPI server

Reply:
Ack. So this has nothing to do with our code.